.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,223 @@
+      - Others (Unknown)
+  validations:
+    required: false
+
+- type: dropdown
+  id: device
+  attributes:
+    label: Device
+    description: What kind of computer is it?.
+    options:
+      - Desktop PC
+      - Server PC
+      - Mobile device (Smartphone, Smartwatch, Tablet)
+      - Docker/Qemu (Container)
+      - VirtualBox/Vagrant (Virtual Machine)
+      - Embedded/IOT
+      - Arduino Kit/ESP32 Kit
+      - Game Console (Playstation, XBOX, SteamDeck, Switch)
+      - SmartTV/SmartDisplay/Smartprojector
+      - Drone/Robot/Exoskeleton
+      - ASIC/FPGA/Crypto-mining hardware
+      - PLC/Industrial/heavy machine
+      - Point Of Sale/Kiosk/ATM
+      - Car/Self-Driving/On-Board Computer
+      - Electric scooter/Electric bike
+      - Satellite/MicroSatellite
+      - Military machine
+      - Others (Unknown)
+  validations:
+    required: false
+
+- type: textarea
+  id: what-happened
+  attributes:
+    label: What happened?
+    description: |
+      Use DETAILED DESCRIPTIVE information about the problem.
+      Here, you go into more details about your Bug report. This section can be a few paragraphs long.
+    placeholder: Bug reports with full repro code and detailed information will be fixed faster.
+  validations:
+    required: true
+
+- type: textarea
+  id: current-logs
+  attributes:
+    label: Current Standard Output Logs
+    description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+    placeholder: Bug reports with full repro code and detailed information will be fixed faster.
+    render: shell
+
+- type: textarea
+  id: expected-logs
+  attributes:
+    label: Expected Standard Output Logs
+    description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+    placeholder: Bug reports with full repro code and detailed information will be fixed faster.
+    render: shell
+
+- type: textarea
+  id: possible-solution
+  attributes:
+    label: Possible Solution
+    description: Propose a possible solution.
+  validations:
+    required: false
+
+- type: textarea
+  id: extra-info
+  attributes:
+    label: Additional Information
+    description: Any additional relevant information.
+  validations:
+    required: false
+
+- type: textarea
+  id: incompatibility
+  attributes:
+    label: Backwards Compatibility
+    description: If your Bug Report introduces backward-incompatible changes, describe them and propose how to deal with them.
+  validations:
+    required: false
+
+- type: markdown
+  attributes:
+    value: |
+      - Thanks for your contributions!, your Bug report will receive feedback from the community soon...
+      - **Remember to :star: Star the Nim project on GitHub!.**
+      - Please check whether the problem still exists in git devel branch, see [rebuilding the compiler](https://nim-lang.github.io/Nim/intern.html#rebuilding-the-compiler).
```

```suggestion
      - Please check whether the problem still exists in the devel branch, see [rebuilding the compiler](https://nim-lang.github.io/Nim/intern.html#rebuilding-the-compiler).
```
Maybe suggest `choosenim devel --latest`, that's a lot easier imo.